integration/tests/posit/connect/test_env.py

```diff
@@ -0,0 +1,41 @@
+from posit import connect
+
+
+class TestEnvVars:
+    @classmethod
+    def setup_class(cls):
+        cls.client = connect.Client()
+        cls.content = cls.client.content.create(
+            name="Sample",
+            description="Simple sample content for testing",
+            access_type="acl",
+        )
+
+    @classmethod
+    def teardown_class(cls):
+        cls.content.delete()
+        assert cls.client.content.count() == 0
+
+    def test_clear(self):
+        self.content.environment_variables.create("KEY", "value")
+        assert self.content.environment_variables.find() == ["KEY"]
```

Comment:
Did you consider a dict-like interface for env vars? IDK if it would make sense, but it would look more like

```
        self.content.environment_variables["KEY"] = "value"
        assert self.content.environment_variables.keys() == ["KEY"]
```

which feels natural to type.

Maybe that could be sugar on top of the CRUD methods you've wrapped here? Maybe it's a bad idea? 🤷

Reply:
Ah nm I see you have these too, just not in the integration tests 👍